src/output/html.js

```diff
@@ -18,7 +18,7 @@ const {
   eventStatuses
 } = require("../shared");
 const { processFantasyResults } = require("../fantasy/fantasyCalculator");
-const { getLocalization } = require("./localization");
+const { getLocalization, translate } = require("./localization");
```

Comment:
It would be really good to only have one way to do the translation (ie: removing the getLocalization method). The bulk of the translation work happens in the handlebars templates and they just access simple fields to do the work. I had a quick look and you can call functions from handlebars by registering a helper - check out https://stackoverflow.com/questions/21845710/calling-javascript-function-from-handlebar. If you're keen and could get that working for your translate method inside handlebars I'll merge it in for sure :)

Reply:
I'll look in to this when i get the time to sit down and experiment with it! :)

Reply:
i'll look into this when i get the time to sit down and mess around with it! :)